newrelic.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable header/header */
```

Comment:
Is there a way we could add to the ignore via the eslintrc instead of a line here? Just cause there's copy-paste going on and maybe in the future might use as starting point for the CLI install workflow.

Looks like maybe that "overrides" section is for that? I've honestly never tried. If too much of a pain we can roll with this.